src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -883,7 +886,7 @@ impl HummockVersion {
             )
         });
 
-        group_split::merge_levels(left_levels, right_levels);
+        group_split::merge_levels(left_levels, right_levels, self.max_sub_level_id);
```

Comment:
Not sure if it's better to maintain a per-compaction group max_sub_level_id instead of a global one, and then we don't need to pass the global one here.

Reply:
I'm open to either option. @Li0k

Reply:
I prefer global uniqueness, for me at the moment this sub level id has no more semantics. So globally unique is more concise.